From 29dbb597089981a5e3c97ae4f768ff8a22094294 Mon Sep 17 00:00:00 2001
From: Bhupesh Sharma <bhsharma@redhat.com>
Date: Thu, 25 Oct 2018 11:48:27 +0530
Subject: [PATCH] [PATCH] Update help text to indicate --mem-usage is supported
 on archs other than x86_64

Commit 8449bda73ab14516d4bf81d29503c1ea203bb865 ("Documentation:
Update documentation regarding --mem-usage' option"), updated the
makedumpfile man page to indicate that this option is now
supported on x86_64, arm64, ppc64 and s390x.

However, the help text for makedumpfile (which one can see
by running 'makedumpfile --help'), still reflects the old
support status, i.e. this option is only supported on x86_64.

This patch changes the help text to be in sync with the man
page text.

Signed-off-by: Bhupesh Sharma <bhsharma@redhat.com>
---
 print_info.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/makedumpfile-1.6.2/print_info.c b/makedumpfile-1.6.2/print_info.c
index 732d5b806c24..0be12ea8803c 100644
--- a/makedumpfile-1.6.2/print_info.c
+++ b/makedumpfile-1.6.2/print_info.c
@@ -300,7 +300,7 @@ print_usage(void)
 	MSG("      necessary to specify [-x VMLINUX] or [-i VMCOREINFO].\n");
 	MSG("\n");
 	MSG("  [--mem-usage]:\n");
-	MSG("      This option is only for x86_64.\n");
+	MSG("      This option is currently supported on x86_64, arm64, ppc64 and s390x.\n");
 	MSG("      This option is used to show the page numbers of current system in different\n");
 	MSG("      use. It should be executed in 1st kernel. By the help of this, user can know\n");
 	MSG("      how many pages is dumpable when different dump_level is specified. It analyzes\n");
-- 
2.7.4

